math3/math3.h

```diff
@@ -270,7 +308,9 @@ struct APIFUNCT
 #include <stdio.h>
 #include <stdlib.h>
 
+#ifdef _WIN32
```

Comment:
I should just get rid of this header and all uses of ``TCHAR``. Better to use explicit W version of Win32 APIs and wchar_t.